docs/usage.rst

```diff
@@ -242,6 +242,86 @@ Thus we see that there are multiple Nash equilibria for this game. Two pure equi
 
 To further study how this system evolves over time and how robust some of the observations we have made are let us look at how this game can be interpreted in an ecological setting.
```

Comment:
This paragraph introduces the ecological tournament. Could you move this to the end of the section on noisy tournaments and then add a sentence here to replace it which introduces noisy tournaments?